scripts/index.js

```diff
@@ -1 +1,229 @@
-// add your solutions here
\ No newline at end of file
+// add your solutions here
+
+// Positive Numbers
+// Write a function which takes an array of numbers as 
+// input and returns a new array containing only the 
+// positive numbers in the given array.
+
+function positiveNumbers(arrayOfNumbers){
+    const positiveArray = arrayOfNumbers.filter(function (number) {
+        return number > 0;
+    });
+    return positiveArray;
+}
+
+/////////////////////////////////////////////////////////////////////////////
+
+// Even Numbers
+// Write a function which takes an array of numbers as 
+// input and returns a new array containing only the 
+// even numbers in the given array.
+
+function evenNumbers(arrayOfNumbers){
+    const evenArray = arrayOfNumbers.filter(function (number){
+        return (number % 2) == 0;
+    });
+    return evenArray;
+}
+
+/////////////////////////////////////////////////////////////////////////////
+
+// Square the Numbers
+// Write a function which takes an array of numbers as 
+// input and returns a new array containing result of 
+// squaring each of the numbers in the given array by two. 
+// Example: squareTheNumbers([1, 2, 3]) should give [1, 4, 9].
+
+function squareTheNumbers(arrayOfNumbers){
+    const squareArray = arrayOfNumbers.map(function (number){
+        return (number * number);
+    });
+    return squareArray;
+}
+
+/////////////////////////////////////////////////////////////////////////////
+
+// Cities 1
+// Write a function which takes an array of city objects like such:
+
+var cities = [
+  { name: 'Los Angeles', temperature: 60.0},
+  { name: 'Atlanta', temperature: 52.0 },
+  { name: 'Detroit', temperature: 48.0 },
+  { name: 'New York', temperature: 80.0 }
+];
+// as input and returns a new array containing the cities 
+// whose temperature is cooler than 70 degrees.
+
+function coolCities(cities){
+    const coolArray = cities.filter(function (city){
+        return city.temperature < 70.0;
+    });
+    const coolCitiesArray = coolArray.map(function (city){
```

Comment:
Nice! Can you think of a way to omit both the `coolArray` and `coolCitiesArray` variables? (Totally fine to leave them in, though.)